io.freetubeapp.FreeTube.metainfo.xml

```diff
@@ -3,16 +3,36 @@
   <id type="desktop-application">io.freetubeapp.FreeTube</id>
   <metadata_license>CC0-1.0</metadata_license>
   <name>FreeTube</name>
-  <summary>An Open Source YouTube app for privacy</summary>
+
+  <summary>An open-source YouTube app for privacy</summary>
+  <summary xml:lang="hu">Nyílt forráskódú YouTube-alkalmazás az adatvédelem érdekében</summary>
+
   <description>
-    <p>FreeTube is an open source desktop YouTube player built with privacy in
-    mind. Use YouTube without advertisements and prevent Google from tracking
-    you with their cookies and JavaScript. Available for Windows, Mac &amp;
-    Linux thanks to Electron.</p>
-    <p>Please note that FreeTube is currently in Beta. While it should work well
-    for most users, there are still bugs and missing features that need to be
-    addressed.</p>
+    <p>
+      FreeTube is an open-source desktop YouTube player built with privacy in
+      mind. Use YouTube without advertisements and prevent Google from tracking
+      you with their cookies and JavaScript. Available for GNU/Linux, macOS, and
+      Windows thanks to Electron.
+    </p>
+    <p>
+      Please note that FreeTube is currently in Beta. While it should work well
+      for most users, there are still bugs and missing features that need to be
+      addressed.
+    </p>
+    <p xml:lang="hu">
+      A FreeTube egy nyílt forráskódú asztali YouTube-lejátszó, amely a magánélet
+      védelmét szem előtt tartva készült. Használja a YouTube-ot hirdetések nélkül,
+      és akadályozza meg, hogy a Google nyomon kövesse Önt a cookie-jaival és a
+      JavaScripttel. Elérhető GNU/Linux, macOS és Windows rendszerekhez az
+      Electronnak köszönhetően.
+    </p>
+    <p xml:lang="hu">
```

Comment:
placement in the same way as reverted version

Reply:
The convention used is also used in [GIMP](https://flathub.org/hu/apps/org.gimp.GIMP). Unfortunately, the exact GIMP file for your reference is unable to be found.